doc/guide_3rd_party.md

```diff
@@ -176,10 +268,73 @@ Specify the amount to transfer to the receiver wallet. If needed the certificate
 
 By default hashed attributes are not included in the transfer. Make sure to include any hashed attributes, in the transfer request, the receiver wallet owner should be able to see.
 
+```csharp
+var token = "<access-token>"; // Access token obtained with client-credentials
+var organizationId = "<organizationId>"; // Organization id of sender
+
+var requestObject = new
+{
+    CertificateId = certificateFederatedStreamId, // the federatedStreamId from the get Certificates endpoint
+    ReceiverId = externalEndpointId, // The externalEndpointId created earlier
+    Quantity = quantity, // How much quantity wanted claimed. (Eg. all of the consumer quantity or whatever you want to trasnfer)
+    HashedAttributes = new string []
+    {
+        "AssetId"
+    }
+};
+
+var httpClient = new HttpClient();
+var request = new HttpRequestMessage(HttpMethod.Post, $"https://demo.energytrackandtrace.dk/wallet-api/transfers?organizationId={organizationId}")
+{
+    Content = new StringContent(JsonSerializer.Serialize(requestObject), Encoding.UTF8, "application/json")
+};
+request.Headers.Authorization = new AuthenticationHeaderValue("bearer", accessToken);
+request.Headers.Add("X-API-Version", "1");
+var response = await httpClient.SendAsync(request);
+
+var transfer = await response.Content.ReadFromJsonAsync<TransferResponse>();
+```
+
 ## Claim Energy
 
 Given a production certificate and a consumption certificate in a wallet, it is possible to claim the produced energy. Use the <https://demo.energytrackandtrace.dk/developer#tag/Claims/paths/~1wallet-api~1claims/post> endpoint to claim two certificates. If needed to certificates will  be sliced automatically to match the amount of quantity that should be claimed.
 
+```csharp
+var token = "<access-token>"; // Access token obtained with client-credentials
+var organizationId = "<organizationId>"; // Organization id owner of both certificates
+
+var requestObject = new
+{
+    ProductionCertificateId = productionCertificateFederatedStreamId,
+    ConsumptionCertificateId = consumptionCertificateFederatedStreamId,
+    Quantity = quantity
+};
+
+var httpClient = new HttpClient();
+HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Post, $"https://demo.energytrackandtrace.dk/wallet-api/claims?organizationId={consumerConsents.OrganizationId}")
+{
+    Content = new StringContent(JsonSerializer.Serialize(requestObject), Encoding.UTF8, "application/json")
+};
```

Comment:
No example response?

Reply:
It's only a ID that we are not using anywhere. It's similar to the Trasnfer API that is also a fire and forget.